client/blocks/eligibility-warnings/style.scss

```diff
@@ -174,9 +218,18 @@
 .eligibility-warnings__warnings-card {
 	background: #f6f7f7;
 	margin: 0 24px;
+
+	.eligibility-warnings__message {
```

Comment:
For the warnings message, add this line at the file `wp-content/lib/atomic/eligibility.php` on line 313 to see them: `$widget_warnings = get_unsupported_widgets();`